internal/npm_install.bzl

```diff
@@ -53,6 +60,10 @@ npm_install = repository_rule(
             mandatory = True,
             single_file = True,
         ),
+        "package_lock_json": attr.label(
+            allow_files = True,
```

Comment:
for future reference - there is a shortcut `allow_single_file` that does both things